applications/asset_tracker_v2/boards/thingy91_nrf9160ns.conf

```diff
@@ -22,3 +22,11 @@ CONFIG_ADXL362_ACCEL_ODR_12_5=y
 
 # BME680 - Temperature and humidity sensor.
 CONFIG_BME680=y
+
+# CAF - Common Application Framework
+CONFIG_GPIO=n
+CONFIG_PWM=y
+CONFIG_LED_GPIO=n
+CONFIG_LED_PWM=y
+CONFIG_CAF_LEDS_GPIO=n
```

Comment:
See comment regarding CONFIG_CAF_LEDS_PWM in the other file. It applies here but in reverse.